### ISSUE TITLE
Feature idea: Free content library

### ISSUE THREAD

**Issue by SubJunk** [UniversalMediaServer/UniversalMediaServer#4921]
We have good support for audio/video feeds and online radio, so I think it would help users if that was more discoverable.

I am imagining a way on the web player to see lists of available online content, and they can click to add it to their UMS Shared Content.

The lists can be provided by our API so we can keep it updated without the user needing to install new versions of UMS.

What do you think @UniversalMediaServer/developers ?

**Comment by ik666** [comment]
Adding content is, generally speaking, a good idea. That's why I've added support for that in #4918.

It would probably be hard to curate such a list ...

**Comment by SubJunk** [comment]
Yes maybe we can find curated service/s to consume via the API like we do with TMDB. Then if the services disappear we can replace them without breaking support for existing versions of UMS

**Comment by SubJunk** [comment]
This seems like a good candidate for radio content https://github.com/nepodev/radio-browser

**Comment by ik666** [comment]
> This seems like a good candidate for radio content https://github.com/nepodev/radio-browser

Yes, the radio browser API is already integrated with the last radio-station playlist support. But there are thousands of radio stations listed ... not really "curated" ...

**Comment by SubJunk** [comment]
They can be ordered by votes so it is user-curated, do you have another suggestion though? I am interested in any ideas, especially since I don't use online radio

**Comment by ik666** [comment]
Letting the user add radio stations by themselves would be one obvious thing that can be done. The radio browser stations could be searched by country, name, popularity, etc ...
